next.config.js

```diff
@@ -21,6 +21,15 @@ const nextConfig = {
   // ? https://nextjs.org/docs/advanced-features/output-file-tracing#automatically-copying-traced-files
   output: 'standalone',
   poweredByHeader: false,
+  async redirects() {
+    return [
+      {
+        source: '/',
+        destination: '/hub',
+        permanent: false,
+      },
+    ];
+  },
```

Comment:
Could you remove this? Finally, the `/` route is called Hub, which may not updated in the technical document.